tools/Az/Az.psd1

```diff
@@ -89,7 +89,7 @@ RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.2.8'; },
                @{ModuleName = 'Az.Kusto'; RequiredVersion = '1.0.1'; }, 
                @{ModuleName = 'Az.LogicApp'; RequiredVersion = '1.5.0'; }, 
                @{ModuleName = 'Az.MachineLearning'; RequiredVersion = '1.1.3'; }, 
-               @{ModuleName = 'Az.Maintenance'; RequiredVersion = '1.1.0'; }, 
+               @{ModuleName = 'Az.Maintenance'; RequiredVersion = '1.2.0'; }, 
```

Comment:
Please roll back this change. It will be updated automatically.

Reply:
Fixed